src/network/connection.cpp

```diff
@@ -2986,7 +2942,7 @@ float Connection::getLocalStat(rate_stat_type type)
 	return retval;
 }
 
-u16 Connection::createPeer(Address& sender, MTProtocols protocol, int fd)
+u16 Connection::createPeer(const Address &sender, size_t sock_id, MTProtocols protocol, int fd)
```

Comment:
Well, it should be a `std::vector<UDPSocket*>::size_type`, but I figured `size_t was close enough.